mri/operators/fourier/orc_wrapper.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+"""
+This modules contains classes for Off-Resonance Correction (ORC)
+
+:Author: Guillaume Daval-Frérot <guillaume.davalfrerot@cea.fr>
+"""
+
+import numpy as np
+import sklearn.cluster as sc
+
+from .utils import (compute_mfi_coefficients,
+                    compute_mti_coefficients,
+                    compute_svd_coefficients)
+
+
+class ORCFFTWrapper():
```

Comment:
Can we add some tests to check the correctness of this wrapper? Something like the output of this and normal FFT is same in absence of resonance artifacts. Note that you can add cpu based tests (which run on CI) and also gpu based tests (which we plan to run in future in CI, but for now can be run locally for checking correctness)